### PR TITLE
Edit multiple words

### PR DESCRIPTION
**Pull request by strelok2012**

### CODE REVIEW

File: app/controller.js [hunk 1]
```diff
@@ -856,6 +856,12 @@ class MainController {
 
         region.element.classList.add("selected-region");
 
+        region.data.words.forEach((w) => {
+            if (!w.uuid) {
```
Comment (GolanLevy) [inline app/controller.js]: I do not understand why do we manipulate data on 'selectRegion' function.
Why not to execute this code while loading the data and every time a new word appears

File: app/utils.js [hunk 2]
```diff
@@ -6,4 +6,11 @@ export const jsonStringify = (json) => {
         }
         return value;
     })
+}
```
Comment (GolanLevy) [inline app/utils.js]: I guess you have copies this code from:
https://stackoverflow.com/questions/105034/create-guid-uuid-in-javascript

Notice that the author says it is not the best solution and node-uuid is better.
I would not change it now but just keep that in mind

File: app/controller.js (hunk 1 above)
Comment (strelok2012) [reply app/controller.js]: There is just in optimization purposes - i.e. for demo file we need 1497 uuids on a loading, I think creating uuids on demand it's a better way

File: app/utils.js (hunk 2 above)
Comment (strelok2012) [reply app/utils.js]: Done, i replaced it with node-uuid, i think we will need it in the future

**Pull request merged**